onnxruntime/core/session/abi_session_options.cc

```diff
@@ -184,3 +184,17 @@ ORT_API_STATUS_IMPL(OrtApis::AddSessionConfigEntry, _Inout_ OrtSessionOptions* o
                     _In_z_ const char* config_key, _In_z_ const char* config_value) {
   return onnxruntime::ToOrtStatus(options->value.AddConfigEntry(config_key, config_value));
 }
+
+ORT_API_STATUS_IMPL(OrtApis::AddInitializer, _Inout_ OrtSessionOptions* options, _In_z_ const char* name_str,
+                    _In_ OrtValue* val) {
+  if (!name_str || !val) {
```

Comment:
>name_str [](start = 7, length = 8)

Do we have a or need a MACRO like 
RETURN_ON_NULL(N, V)  if (!V) { return OrtApis::CreateStatus(ORT_INVALID_ARGUMENT, std::format(...)); }
?
Combining the checks and error messages on multiple variables can be very confusing on APIs.